cylc/flow/scripts/message.py

```diff
@@ -105,7 +105,12 @@ def main(parser, options, *args):
         return parser.error('No message supplied')
     cylc.flow.flags.verbose = os.getenv('CYLC_VERBOSE') == 'true'
     cylc.flow.flags.debug = os.getenv('CYLC_DEBUG') == 'true'
-    if len(args) <= 2:  # compat
+    if len(args) <= 2:
+        # BACK COMPAT: args <= 2
+        # from:
+        #     ????
+        # remove at:
+        #     ????
```

Comment:
Since the `cylc message -- xyz` interface is this still considered legacy?

Reply:
The new interface was introduced 3 years ago (cylc-7.6-ish?). I don't mind if we ditch the back-compat now, but there is a pretty high chance the "class interface" is being used somewhere.  Note we still use it ourselves and some tests.

```
$ git grep "cylc message" | grep -v '\-\-' | wc -l
35
```

Reply:
Think I'll leave it as `???` and kick this down the road.